configure.py

```diff
@@ -1090,6 +1090,7 @@ def find_ninja():
                 'locator/util.cc',
                 'service/client_state.cc',
                 'service/storage_service.cc',
+                'service/session.cc',
```

Comment:
CMakeLists.txt

Reply:
I think the name session is too generic. Better to have a more specific name, e.g.,  global_work_sessions. The name is longer but it is more clear what the session is about.

Reply:
In addition, we already have streaming/stream_session.cc.  It is too confusing to just call it session which will be mainly used in the stream_sessions.